core/domain/rights_manager.py

```diff
@@ -180,22 +208,50 @@ def _save_activity_rights(
     model.commit(committer_id, commit_message, commit_cmds)
 
 
-# Update summary of changed activity (note that the activity rights id is the
-# same as the activity id).
-# TODO(msl): get rid of inline imports by refactoring code
+# TODO(msl): get rid of inline imports by refactoring code.
 def _update_exploration_summary(activity_rights):
+    """Updates the exploration summary for the given activity(associated with
```

Comment:
Nit: add space before '('. Ditto below.

Actually, perhaps it would be better to write:

    Updates the exploration summary for the activity associated with the given rights object.

since there isn't actually a "given activity" in this case. (Ditto below.)